pkgs/development/python-modules/sphinx-pallets-theme/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, sphinx, packaging }:
+
+buildPythonPackage rec {
+  pname = "Pallets-Sphinx-Themes";
```

Comment:
Can you normalize `pname` to `sphinx-pallets-theme`? Like the attribute and directory is called.

Reply:
Fixed.

Reply:
The pname should be `pallets-sphinx-theme`, see https://github.com/pallets/pallets-sphinx-themes/blob/main/setup.py#L4 and 

> Attribute names in python-packages.nix as well as pnames should match the library's name on PyPI, but be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names). This means that characters should be converted to lowercase and . and _ should be replaced by a single - (foo-bar-baz instead of Foo__Bar.baz). If necessary, pname has to be given a different value within fetchPypi.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-contributing

Reply:
I changed `pname` in Nix expression to be lowcase and dashes; I don't see in manual requirement to patch `setup.py`. Or I misunderstand you?

Reply:
The value of the name field in the call to setup is the name on pypi.

Reply:
You want me to patch `setup.py`, like this?https://github.com/NixOS/nixpkgs/pull/172731/commits/06d98e1f073d6bc36207b82fbe38e4fa1cba974d

It will break reverse dependencies. I already lower-cased pname, directory name and attribute name.

Reply:
No, the pname is what needs to be fixed, no patching required.

Reply:
You sure we are looking at same code? I already lower-cased pname, so it looks like most of the stuff in `pkgs/development/python-modules`. Screenshot attached

Reply:
The commit message is still wrong, and so are the attribute name and the directory name.